app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,28 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+@include govuk-media-query($until: tablet) {
+  .govuk-breadcrumbs__list {
+    display: flex;
+    padding: 14px 0;
```

Comment:
Is 14px a magic number? Why 14px?

Reply:
because it makes the touch target 44px high as per the design doc in the trello card

Reply:
It makes the list item 44px high, but the thing the user clicks on is the link, which is still the same height.

Reply:
what do you suggest to ensure a minimum clickable area of 44px ?

Reply:
Increase the height of the link somehow? Padding should do it.

Are you able to test this on an actual mobile device? I can browserstack it from here but it's not the same.

Reply:
added padding to the link. No haven't been able to use real device, checked on browserstack and looks ok though

Reply:
I think it's important to check with a real device if possible, testing with browserstack for touch points isn't the same as the real thing.

Reply:
Tested on my phone, looks ok 👍 There are some iphones in the empathy lab on the 7th floor I think, if you want to test @RobBond-GDS